cmd/gdsutil/main.go

```diff
@@ -405,6 +411,15 @@ func ldbGet(c *cli.Context) (err error) {
 			return cli.Exit(err, 1)
 		}
 
+		if c.Bool("honu") {
+			// Extract the data from the honu metadata
+			obj := &object.Object{}
+			if err = proto.Unmarshal(data, obj); err != nil {
+				return cli.Exit(err, 1)
+			}
+			data = obj.Data
+		}
```

Comment:
A quick helper to get `gdsutil` to parse `honu` data so that I could read what the certificate request was.